tests/test_kernels.py

```diff
@@ -243,7 +235,7 @@ def test_coregion_slice(N, input_dim, output_dim, rank):
 kernel_setups_extended = kernel_setups + [
     SquaredExponential() + Linear(),
     SquaredExponential() * Linear(),
-    SquaredExponential() + Linear(ard=True, variance=rng.rand(_dim, 1).reshape(-1))
+    SquaredExponential() + Linear(variance=rng.rand(_dim, 1).reshape(-1))
```

Comment:
Why not
```suggestion
    SquaredExponential() + Linear(variance=rng.rand(_dim))
```
?

Reply:
Ha - yeah I'm not sure why that was defined that way. I'll change it.